contribute.mdx

```diff
@@ -7,88 +7,124 @@ content:
   paragraph: This page explains how to participate in the Scaleway Write for the Community program.
 tags: write-for-the-community community contribute git voucher
 cta: false
-dates:
-  validation: 2023-10-02
-  posted: 2022-03-24
 ---
 
-<Lightbox src="scaleway-write-for-community-2x.webp" alt="" size="large" hero />
+Join the Scaleway **Write for the Community** program, a collaborative initiative for making our [open-source Documentation](https://github.com/scaleway/docs-content/) a truly collaborative knowledge center.
```

Comment:
"Collaborative" twice - maybe get rid of the first one?

Reply:
Good catch, thanks!